src/components/sidebar/MenuMap.js

```diff
@@ -43,26 +43,28 @@ class MenuMap extends Component {
         </span>
         { !/Benchmarks/.test(item.title) &&
         <div className="sidebar--menu--detail--button--container">
-          <span
+          <FileText
             className={`sidebar--menu--detail--button solution ${this.props.theme} cm-variable`}
             id={'SOLUTION__' + id}
-            onClick={this.selectSolution}>
-            Solution
-          </span>
-          <span
+            data-tip
+            data-for='solutionTip'
+            onClick={this.selectSolution} />
```

Comment:
☝️ not sure what's going on here with the solution button not working, I'd have to take a closer look. But the connection should have nothing to do with it, there's no real service being called here, it's all client side with redux, so something else is up. I'll know more once I pull it down.